spec/features/user_changes_password_spec.rb

```diff
@@ -0,0 +1,32 @@
+feature "User changes password" do
+  before :each do
```

Comment:
I've updated this, but I find myself wanting to DRY it up a bit since every test needs exactly the same entry path.